### PR TITLE
(update) URL Structure for Checkout and Multi Currency Support (Part 2/2)

### PR DESCRIPTION
**Pull request by thisisayush**
This PR is a part of https://github.com/blockonomics/whmcs-bitcoin-plugin/pull/141

- Update Templates to Latest Version from Woocommerce
- Update Order.css to Latest Version from Woocommerce
- Update and refactor app.js to Latest Version from Woocommerce
- Add Crypto Fonts
- Update Blockonomics.php to add and refactor new changes for Templates

### CODE REVIEW

File: modules/gateways/blockonomics/assets/js/app.js [hunk 1]
```diff
@@ -1,200 +1,196 @@
...
-            Order.get({
-                "get_order": $scope.order_uuid,
-                "crypto": $scope.currency.code
-            }, function(data) {
-                $scope.spinner = false;
-                $scope.order_id = data.id_order;
-                
-                if(data.txid !== undefined && data.txid !== ""){
-                    $scope.txid = data.txid;
-                    $scope.pending_error = true;
```
Comment (DarrenWestwood) [inline modules/gateways/blockonomics/assets/js/app.js:121]: Pending transactions are returned in WHMCS to ensure a customer only receives a 2nd payment address after the 1st transaction is confirmed when paying for a single invoice with multiple transactions: https://github.com/blockonomics/whmcs-bitcoin-plugin/blob/master/modules/gateways/blockonomics/blockonomics.php#L512-L515

File: modules/gateways/blockonomics/assets/templates/crypto_options.tpl [hunk 2]
```diff
@@ -2,24 +2,17 @@
...
-      {foreach $active_currencies as $code=>$crypto}
-        <tr class="bnomics-select-options">
-          <td align="left">
-            <a href="{$WEB_ROOT}/modules/gateways/blockonomics/payment.php?show_order={$order_hash}&crypto={$code}" style="color: inherit; text-decoration: inherit;">
-              <img src="{$WEB_ROOT}/modules/gateways/blockonomics/assets/img/{$code}.png" class="rotateimg{$code}" alt="{$code} Logo">
-              <h3>{$crypto['name']}</h3>
-              <span class="bnomics-select-currency-button">
-                <button 
-                  type="button"
-                  class="btn btn-lg bnomics-select-currency-code"
```
Comment (DarrenWestwood) [inline modules/gateways/blockonomics/assets/templates/crypto_options.tpl:16]: Let's reinstate btn and btn-lg classes to improve the appearance with the default whmcs theme.

File: modules/gateways/blockonomics/assets/templates/checkout.tpl [hunk 3]
```diff
@@ -21,25 +18,25 @@
...
+            <!-- Gap limit + Duplicate Address Error -->
+            <div id="address-error-message" ng-show="customer_facing_error" ng-cloak>
+                <h2>{$_BLOCKLANG.error.addressGeneration.title}</h2>
+                <p>[[customer_facing_error]]</p>
             </div>
             <!-- Payment Expired -->
             <div class="bnomics-order-expired-wrapper" ng-show="order.status == -3" ng-cloak>
                 <h3 class="warning bnomics-status-warning">{$_BLOCKLANG.paymentExpired}</h3><br>
-                <p><a href="#" ng-click="try_again_click()">{$_BLOCKLANG.tryAgain}</a></p>
+                <p><a ng-click="try_again_click()">{$_BLOCKLANG.tryAgain}</a></p>
```
Comment (DarrenWestwood) [inline modules/gateways/blockonomics/assets/templates/checkout.tpl]: The try again link does not show cursor pointer on hover. Let's fix this behavior.

Comment (DarrenWestwood) [inline modules/gateways/blockonomics/assets/templates/checkout.tpl:39]: The try again link does not show cursor pointer on hover. Let's fix this behavior.

**Comment by thisisayush** [comment]
Merged 5275eba82747c52829bb6999f98b4fb185f36fb9 and Closed

**Pull request closed without merging**